### PR TITLE
docs: add groth16

### PR DESCRIPTION
**Pull request by mattstam**
Mention Groth16 as an alternative to PLONK.

### CODE REVIEW

File: book/generating-proofs/recommended-workflow.md [hunk 1]
```diff
@@ -40,18 +40,18 @@ Note that **latency is not the same as cost**, because we parallelize proof gene
...
+- An average Ethereum block can be between 100-500M cycles (including merkle proof verification for storage and execution of transactions) with our `keccak` and `secp256k1` precompiles.
+- For a Tendermint light client, the average cycle count can be between 10M and 50M cycles (including our ed25519 precompiles).
+- We consider programs with <2M cycles to be "small" and by default, the fixed overhead of proving will dominate the proof latency. If latency is incredibly important for your use-case, we can specialize the prover network for your program if you reach out to us.
 
-Note that if you generate PLONK proofs on the prover network, you will encounter a fixed overhead of 90 seconds for the STARK -> SNARK wrapping step. We're actively working on reducing this overhead in our next release.
+Note that if you generate Groth16 or PLONK proofs on the prover network, you will encounter a fixed overhead of 90 seconds for the STARK -> SNARK wrapping step. We're actively working on reducing this overhead in our next release.
```
Comment (mattstam) [inline book/generating-proofs/recommended-workflow.md:49]: note: not sure if this 90 seconds comment is still relevant

Comment (ctian1) [reply book/generating-proofs/recommended-workflow.md:49]: I think it's lower than that, and even lower in v3.

Comment (mattstam) [reply book/generating-proofs/recommended-workflow.md:49]: seems to vary between v2 and v3, I can just remove this comment

**Review by ctian1 — APPROVED** [review]

File: book/getting-started/hardware-requirements.md [hunk 2]
```diff
@@ -30,13 +31,12 @@ which can be parallelized with multiple cores.
...
-the memory requirements of the PLONK prover substantially.
+This effect is most noticeable when using the Groth16 or PLONK provers, which requires around 128GB
+of RAM to generate a proof.
 
 ### Disk
 
-Disk is required to install the SP1 zkVM toolchain and to install the trused setup artifacts, if you plan to locally build the PLONK prover.
-
-Furthermore, disk is used to checkpoint the state of the program execution, which is required to generate the proofs. 
+Disk is required to install the SP1 zkVM toolchain and to install the trused setup artifacts, if you
```
[the remark this answers is not in the document]
Comment (mattstam) [reply book/getting-started/hardware-requirements.md:39]: updated

File: book/getting-started/hardware-requirements.md [hunk 3]
```diff
@@ -12,16 +13,16 @@ If you want to generate SP1 proofs locally, this section has an overview of the
...
 **The most important requirement is CPU for performance/latency and RAM to prevent running out of memory.**
 
-|                | Mock / Network               | Core / Compress                    | PLONK (EVM)                |
-|----------------|------------------------------|------------------------------------|----------------------------|
+|                | Mock / Network               | Core / Compress                    | Groth16 and PLONK (EVM)    |
+| -------------- | ---------------------------- | ---------------------------------- | -------------------------- |
 | CPU            | 1+, single-core perf matters | 16+, more is better                | 32+, more is better        |
-| Memory         | 8GB+, more is better         | 32GB+, more if you have more cores | 64GB+ (for PLONK)         |
+| Memory         | 8GB+, more is better         | 32GB+, more if you have more cores | 64GB+, more is better      |
 | Disk           | 20GB+                        | 20GB+                              | 100GB+ (for trusted setup) |
```
[the remark this answers is not in the document]
Comment (mattstam) [reply book/getting-started/hardware-requirements.md:20]: updated

**Pull request merged**